tests/phpunit/tests/functions/isBlogInstalled.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * @since 6.0.0
+ *
+ * @group functions.php
+ * @covers ::is_blog_installed
+ */
+class Tests_Functions_IsBlogInstalled extends WP_UnitTestCase {
+
+	/**
+	 * Tests the default return value of is_blog_installed().
+	 * 
+	 * @ticket 54754
+	 */
+	public function test_is_blog_installed() {
+		$this->assertTrue( is_blog_installed() );
+	}
+
+	/**
+	 * @ticket 54754
+	 *
+	 * the other routes to true
+	 */
+	public function test_is_blog_installed_cache() {
+		// Prime the cache with true.
+		wp_cache_set( 'is_blog_installed', true );
+		$this->assertTrue( is_blog_installed(), 'cache set to true' );
+		// Prime the cache with false.
+		// we still get true as it then looks at the value in options table.
+		wp_cache_set( 'is_blog_installed', false );
+		$this->assertTrue( is_blog_installed(), 'cache set to false' );
+
+		// empty the siteurl
```

Comment:
```suggestion
		// Skip an early return when `siteurl` is set.
```
Inline comments should:
- Begin with an uppercase letter.
- End with a period `.`.
- Clearly explain _why_ the following line is needed.

🔢 Applies to other instances in the PR.